src/helpers/validateEmail.ts

```diff
@@ -0,0 +1,14 @@
+export default (email: string) => {
```

Comment:
Works, but we already have email validation in `useProfileEditEmail.ts`. It's much simpler, do you think you could refactor it so `validateEmail.ts` does that instead? It would simplify this helper function.

Reply:
Thanks for the feedback! I just push a new commit implementing this change. I've refactored validateEmail.ts to simplify the helper function as suggested. Let me know if there's anything else I can help with.
Also, I noticed that the hook useProfileEditEmail is not being used.
Let me know if there's anything else I can help with!

Reply:
@alexxispn Thanks for letting me know, I'll make a note to check that. Thanks!